etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/metadata/PartnerMetadata.java

```diff
@@ -35,16 +36,25 @@ public PartnerMetadata(
             String sender,
             Instant timeReceived,
             String hash,
-            PartnerMetadataStatus deliveryStatus) {
-        this(receivedSubmissionId, null, sender, null, timeReceived, hash, deliveryStatus);
+            PartnerMetadataStatus deliveryStatus,
+            String failureReason) {
+        this(
+                receivedSubmissionId,
+                null,
+                sender,
+                null,
+                timeReceived,
+                hash,
+                deliveryStatus,
+                failureReason);
```

Comment:
I would actually not pass in `failureReason` to this constructor and then pass in `null` into the default constructor.

Reply:
I'm not 100% clear on this one; Am I changing the `failureReason` on L49 to `null`? Or do I need to go further up in this class?